chapters/chap03-sec02.tex

```diff
@@ -18,17 +18,97 @@ \subsection{\mimc{} Encryption}\label{instantiation:mkhash:mimc-encryption}
+
+Likewise, in~\cite[Section 5.1]{albrecht2016mimc} the authors claim that, provided that the cubic map is a permutation over the prime field of interest, then $\mimc{}$ can be used to operate over prime fields of odd characteristic. In this case too, $S(x) = x^3$ is an APN, provided $p \neq 3$ (as reported by~\cite[Theorem 3, item 3]{DBLP:journals/tit/HellesethRS99}).
+
+Furthermore, while~\cite[Section 5.3]{albrecht2016mimc} proposes to relax the choice of the map degree to be of the general form $2^t \pm 1$, so long as the map remains a permutation, the authors showed that, out of these ``exponent shapes'', the case $\exponent{} = 2^t + 1$ is, unfortunately, not as good as it initially seems, in $\FFx{2^n}$, due to term cancellation in fields of characteristic 2 that renders the resulting polynomial ``sparse''\footnote{Since the round function is a polynomial, the whole scheme can be seen as a polynomial which overall degree and ``sparsity'' depend on the underlying field characteristic, degree of the round function and number of rounds.} (the degree of the polynomial will be bounded by $3^r$, $r$ being the number of rounds, which does not constitute an improvement on the status quo (i.e. $\exponent{} = 3$)). For this very reason, exponents of the form $2^t + 1, t > 1$, may not be of interest (sparse polynomial and more expensive arithmetic in the round function).
+
+Likewise, if the map degree $\exponent{}$ is chosen to be of the form $2^t + 1$, with $\gcd(\exponent{}, 2^n - 1) = 1$ in the context of $\mimc{}$ over $\FFx{2^n}$, it is necessary to bear in mind that, without the extra requirement that $t$ needs to be coprime with $n$, then this case is not covered by~\cite[Theorem 2]{DBLP:journals/tit/HellesethRS99}, and $S(x) = x^\exponent{}$ does not have differential 2-uniformity anymore - violating the claim made in~\cite[Section 4.2]{albrecht2016mimc} paragraph ``Linear Attacks'' about optimal resistance against linear and differential cryptanalysis. (In fact, depending on the value of $g = gcd(n, t)$, the map $S(x) = x^{2^t + 1}$ would be differentially $2^g$-uniform~\cite{DBLP:conf/eurocrypt/Nyberg93} - contrasting with the setting considered in paragraph ``Linear attacks'' where $\exponent{} = 3$).
+The case $\exponent{} = 2^t - 1$ does not yield an APN over $\FFx{2^n}$ either (except in the case $t = 2$ which reduces to the case $2^{t'} + 1$ for $t' = 1$). Similar observations show that picking round function degrees of the form $2^t \pm 1$ in the context where $\mimc{}$ is defined over prime fields $\FFx{p}$, $p$ odd prime, does not yield APNs.
+
+\medskip
+
+Overall, when studying the resistance of $\mimc{}$ against statistical attacks, we are interested in the probability that an input difference ($d$) is mapped into an output difference ($D$). That is, we are studying the probability that:
+\[
+    F(x + d) - F(x) = D
```

Comment:
What's F?

Reply:
It simply represents a generic function here. If we want to study the proba that an input diff gets mapped to an output diff on a single round, then F = S (the round function), if we want to study this proba for the whole (resp. several rounds) cipher, F = poly that represents the cipher (resp. the poly that represents a few rounds of the cipher) etc.
Thanks for pointing that out though. If I forgot to explain what `F` was, that's definitely missing!